tools/docker/dev_container.Dockerfile

```diff
@@ -0,0 +1,25 @@
+#syntax=docker/dockerfile:1.1.5-experimental
+FROM tensorflow/tensorflow:2.1.0-custom-op-ubuntu16 as dev_container_cpu
```

Comment:
Here you lost the build ARGS to prepare the switch between cpu and gpu that was in https://github.com/seanpmorgan/addons/pull/14/files#diff-76fe3c36c889ccfcf62f6e35439d030fR3-R5